lib/util/util_unlink.c

```diff
@@ -34,4 +34,11 @@ int util_unlink(const char * filename) {
  return unlink(filename);
 }
 
+#else
+
```

Comment:
Maybe we could just do an #error instead? To avoid it compiling all together if unlink is missing?